### DISCUSSION TITLE
Audio files processing but confidence low for hours. Reboot "fixes" it.

### DISCUSSION THREAD

**Discussion by GeekNJ** [mcguirepr89/BirdNET-Pi#482] [Q&A]
This is related to https://github.com/mcguirepr89/BirdNET-Pi/discussions/427 but there might be 2 different situations occurring.

The main discussion in https://github.com/mcguirepr89/BirdNET-Pi/discussions/427 was audio files stopped being analyzed. That happened to me once, for the first time, over this past weekend. When I noticed this, I restarted the pi and then birdnet-pi started to process all the previously recorded files, which took a few hours. This seems to be 1 situation. 

The second situation I've experienced twice, is that none of the bird calls are receiving a high enough match %. This has happened to me twice in the last couple of days after not occurring for weeks prior. What happens is I notice the overview web page is showing no new observations. I checked View log and it was processing current recordings but all coming back with very low confidence. When I listen to the spectogram, it's non-stop birds that the system has identified with high confidence in the past. 

At 2pm ET I rebooted the pi. Prior to rebooting, it showed the last confidence over 75% (my threshold) at 9:57am. Within 2 mins of rebooting, I got a > 75% confidence hit after nothing for 4 hours.  Its been 10 mins and only that 1 high confidence has been logged despite constant bird chatter being recorded.

Just to recap - I could hear the birds in live/spectogram and they were plentiful. Nothing > 75% for 4 solid hours until I rebooted. The audio files during that 4 hours were being processed. Within a couple mins after reboot, there was a hit > 75%. 

Is the confidence rate handled externally? Any idea why it would seem to be processing but all confidence come back very low? Same setup, same location. 

Any thoughts? If it helps, you can see activity at http://levined.duckdns.org:8080 

David

**Comment by DD4WH** [comment]
@GeekNJ 
David, 
thanks for letting us participate in your BirdNET-Pi audio!
I had a look at your daily plots and listened to your audio. Just some observations from my side:
* the daily plot have very few detections, especially in the middle of the day. This may be caused by the scarcety of birds singing at this time of the year (in the northern hemisphere), but my speculation is, it is also caused by your audio quality
* Check your microphone: the audio has terrible and very loud noise in it. Your should check the recommendations on using microphones and maybe consider buying one of the recommended mics or even better solder one by yourself
* I would speculate that at least part of the problem you report is related to your very noisy audio

**Reply by GeekNJ** [reply]
Thanks. I'm using a inexpensive USB microphone I already had available. The terrible & loud noise is the constant "hum" you are referring to? Is that something that can be reduced via gain/settings in the app or pi while a more permanent solution is acquired?

I imagine this is the thread to read through a couple times - https://github.com/mcguirepr89/BirdNET-Pi/discussions/39 ?

Thanks again @DD4WH

**Reply by DD4WH** [reply]
It is difficult to give advice, if we have only little info on your setup. Could you post a photograph of your mic, cable leading to the Pi etc. What is the model of the mic, do you use a long USB cable etc. A photo would be very helpful. 

* I am pretty certain that the noise cannot be influenced by any software setting or RPi setting, but is produced by the mic/USB hardware
* Do you use the original RPi power supply ? If not, maybe you can try that
* Try to put the microphone capsule as far away from the RPi as possible, consider using a USB extension cable, maybe try that out, if you have one lying around?
* Try to use one of the recommended mic setups in the thread you mentioned

Good luck!

**Reply by GeekNJ** [reply]
Thanks again for the thorough reply. The mic I had available is https://www.amazon.com/dp/B0869R7QPK with the USB cable plugged direct into the RPi. I do have a USB extension cable to try out later today. The mic is about 2 ft away from the pi. 

The power supply on this RPi is one of the official power supplies. It is plugged into a heavy duty extension cord as the location is not close to a power outlet. 

I will work on improving the mic setup. I'm looking to see if https://micbooster.com/clippy-and-pluggy-microphones/65-clippy-em172-microphone.html is available in the US somewhere. I can solder but wouldn't mind an off-the-shelf solution. 

Is there an existing installation available on the web that has is using the em172 or the home brewed one to experience the low noise level for myself?

**Reply by ehpersonal38** [reply]
Patrick uses that capsule or one of equivalent noise level, I think:

https://virginia.birdnetpi.com/index.php?filename=House_Sparrow-90-2022-07-01-birdnet-10:21:37.mp3
https://virginia.birdnetpi.com/index.php?filename=Fish_Crow-100-2022-07-02-birdnet-06:41:37.mp3
https://virginia.birdnetpi.com/index.php?filename=House_Finch-99-2022-06-15-birdnet-11:15:32.mp3

**Reply by mcguirepr89** [reply]
>Patrick uses that capsule or one of equivalent noise level, I think:

@DD4WH is my guide!! I use hardware of his design and craft ~~ highly recommended

**Reply by GeekNJ** [reply]
I have horrible hearing so I really appreciate any advice on sounds/noise.I acquired a UGreen USB and an inexpensive TRRS lavalier mic and it seems quieter than the USB mic I had lying around. The UGreen is connected to a USB extension cable. These are the links to Amazon for what I purchased:

https://www.amazon.com/dp/B08Y8CZB2S - UGREEN USB to 3.5mm Audio Jack, USB A Sound Card Adapter Support Mic TRRS
https://www.amazon.com/dp/B016C4ZG74 - PoP voice Professional Lavalier Lapel Microphone Omnidirectional Condenser Mic

The RPi is capturing much more with the above setup. Thank you for the advice.

**Reply by ehpersonal38** [reply]
Have you tried following this wiki page? https://github.com/mcguirepr89/BirdNET-Pi/wiki/Adjusting-your-sound-card

Different microphones usually need different gain settings
